src/pyggdrasil/distances/_scphylo_wrapper.py

```diff
@@ -0,0 +1,63 @@
+"""Thin wrappers around scPhylo distance and similarity metrics."""
+import scphylo
```

Comment:
Do we have `scphylo` directly as a dependency? - I thought we only have your modified fork `scphylo-tools`. Tried to run it locally and it fails.

Reply:
`scphylo-tools` repository hosts Python package which is imported as `scphylo`. I guess this convention was taken from https://scvi-tools.org/